### PR TITLE
Upgrade to remark-stringify@9

### LINKED ISSUE #18: Upgrade to remark-stringify@9
Some tests started to fail after upgrading remark-parse to latest version by Snyk (there were some vulnerabilities). Looks like remark-stringify@8 and remark-parse@9 are not fully compatible, so the first one should be upgraded as well which leads to internal changes of this library.

### PR DESCRIPTION
**Pull request by 72636c**
This is required to land the security upgrade in #16.

Closes #18.

### CODE REVIEW

File: __test__/slackify-markdown.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ test('Simple text', () => {
 });
 
 test('Escaped text', () => {
-  expect(slackifyMarkdown('*h&ello>world<')).toBe('*h&amp;ello&gt;world&lt;\n');
+  expect(slackifyMarkdown('*h&ello>world<')).toBe('\\*h&amp;ello&gt;world&lt;\n');
```
Comment (72636c) [inline __test__/slackify-markdown.test.js:10]: `remark-stringify@9` seems to be aggressive in escaping some entities, [like this `*`](https://github.com/syntax-tree/mdast-util-to-markdown/blob/a57fe5cc57fb6ef1162962033e55cbb31991418f/lib/unsafe.js#L53). I'll look into whether this causes any issues in Slack.

File: __test__/slackify-markdown.test.js [hunk 2]
```diff
@@ -34,7 +34,7 @@ test('Italic', () => {
 
 test('Bold+Italic', () => {
   const mrkdown = '***bold+italic***';
-  const slack = `${zws}*${zws}_bold+italic_${zws}*${zws}\n`;
+  const slack = `${zws}_${zws}*bold+italic*${zws}_${zws}\n`;
```
Comment (72636c) [inline __test__/slackify-markdown.test.js:37]: The order has changed here, but it doesn't make a difference: https://app.slack.com/block-kit-builder/#%7B%22blocks%22:%5B%7B%22type%22:%22section%22,%22text%22:%7B%22type%22:%22mrkdwn%22,%22text%22:%22_*this*_%20is%20*_fine_*%22%7D%7D%5D%7D

File: __test__/slackify-markdown.test.js (hunk 1 above)
Comment (72636c) [reply __test__/slackify-markdown.test.js]: Looks like Slack `invalid_payload`s an escaped asterisk `\*`, so I've skipped the [`safe` escaping](https://github.com/syntax-tree/mdast-util-to-markdown/blob/main/lib/util/safe.js) for now.

**Comment by jsarafajr** [comment]
This is huuuge @72636c !! 

Thanks for taking the time to rewrite it! 10x 👍

**Comment by jsarafajr** [comment]
Published as `slackify-markdown@v4.0.0`

**Pull request merged**